src/features/runner.ts

```diff
@@ -14,33 +13,33 @@ export class AnsiblePlaybookRunProvider {
   }
 
   /**
-     * Register a set of command callbacks for executing Ansible playbooks
-     * within VS Code.
-     */
+   * Register a set of command callbacks for executing Ansible playbooks
+   * within VS Code.
+   */
   private configureCommands() {
     this.vsCodeExtCtx.subscriptions.push(
       vscode.commands.registerCommand(
         'extension.ansible-playbook.run',
-        fileObj => this.makeCmdRunner()(fileObj),
-      ),
+        (fileObj) => this.makeCmdRunner()(fileObj)
```

Comment:
The braces only add clutter, don't add them unnecessarily.

Reply:
Watch https://github.com/ansible/vscode-ansible/pull/407 for an answer. Outside our control.